packages/virtual/debug/package.mk

```diff
@@ -18,7 +18,7 @@ if [ "$VDPAU_SUPPORT" = "yes" -a "$DISPLAYSERVER" = "x11" ]; then
 fi
 
 if [ "$VAAPI_SUPPORT" = "yes" ]; then
-  PKG_DEPENDS_TARGET="$PKG_DEPENDS_TARGET libva-utils"
+  PKG_DEPENDS_TARGET="$PKG_DEPENDS_TARGET libva-utils vadumpcaps"
```

Comment:
Please remove this change. If you want to add the package that's fine but it's not included in the default build just so one person can use it. If you need it in your build you can add it to your depends or just copy the binary to storage.

Reply:
👍